src/doom/r_things.c

```diff
@@ -782,7 +782,7 @@ void R_SortVisSprites (void)
     int			count;
     vissprite_t*	ds;
     vissprite_t*	best;
-    vissprite_t		unsorted;
+    static vissprite_t		unsorted;
```

Comment:
Fix whitespace.

Reply:
How exactly? Remove one tab in between?

Reply:
Looks likes there are no complains regarding `unsorted`, I see only about `fixed_t bestscale` (is not initialized).

Reply:
There's still a linter message if you view the changes tab in this PR but good enough.